packages/website/src/css/prism.css

```diff
@@ -0,0 +1,160 @@
+:root {
+  --token-color: #393a34;
+  --token-background: #f6f8fa;
+  --token-border: var(--ifm-color-emphasis-300);
+
+  --token-color-comment: #999988;
+  --token-color-doctype: #008000;
+  --token-color-string: #a31515;
+  --token-color-operator: #393a34;
+  --token-color-symbol: #36acaa;
+  --token-color-number: #098658;
+  --token-color-keyword: #0000ff;
+  --token-color-function: #000000;
+  --token-color-inserted: #36acaa;
+  --token-color-deleted: #d73a49;
+  --token-color-important: #e90;
+  --token-color-class-name: #2b91af;
+  --token-color-selector: #800000;
+  --token-color-regexp: #800000;
+  --token-color-property: #ff0000;
+  --token-color-builtin: #0000ff;
+  --token-color-boolean: #0000ff;
+
+  --token-color-highlight: rgba(193, 222, 241, 0.2);
+}
+
+html[data-theme='dark'] {
+  --token-color: #f8f8f2;
+  --token-background: var(--ifm-background-surface-color);
+  --token-border: #414458;
+
+  --token-color-comment: #737373;
+  --token-color-doctype: #6a9955;
+  --token-color-string: #6a9955;
+  --token-color-operator: #d4d4d4;
+  --token-color-symbol: #b5cea8;
+  --token-color-number: #b5cea8;
+  --token-color-keyword: #569cd6;
+  --token-color-function: #569cd6;
+  --token-color-inserted: #50fa7b;
+  --token-color-deleted: #ff5555;
+  --token-color-important: #569cd6;
+  --token-color-class-name: #4ec9b0;
+  --token-color-selector: #d7ba7d;
+  --token-color-regexp: #d7ba7d;
+  --token-color-property: #ce9178;
+  --token-color-builtin: #569cd6;
+  --token-color-boolean: #569cd6;
+
+  --token-color-highlight: rgba(247, 235, 198, 0.2);
+}
+
+div[class*='codeBlockContainer'],
+div[class*='codeBlockContainer'] pre {
```

Comment:
@Josh-Cena i think we should create ticket for this in docosaurus

Reply:
Ah, so we inlined Prism CSS in the CSS bundle rather than using client modules? That looks nice, but does that mean we lose customizability? Do you mean to create a ticket to backport this change in our theme?

Reply:
no, I'm just asking about stable selector, currently there is no way to stable select `codeBlockContainer`, i'm using `*=` as name differs between local/server etc. in prod mode `codeBlockContainer_nb3`

Reply:
Oh, I see, we should have stable class names for code blocks. I would open a PR